File: PYTHON/Daily Flash/Week2/Day1_Jan20/code/prog2.py
'''
Program 2 : Write a Program that accepts an integer from user and print Sum of
all number up to entered number .
Input: 10
Output: The s um number up to 10 : 55
'''

num = int(input("Enter num: "))
sum = 0;
for x in range(num):
        sum +=x+1;      #add

print("The sum of number up to 10: "+str(sum))

File: JAVA/Daily Flash Codes/Week3/Day2_Jan28/code/prog2.java
/*
 * Program 2: Write a Program that accepts Two integers from user and prints the
series of factorial of all numbers between those two inputs.
Input: 1 to 5
Output:
Factorial of 1 = 1
.
.
Factorial of 5 = 120
*/

import java.util.*;

class Program {
	public static void main(String args[]) {
			
		Scanner sc = new Scanner(System.in);
		int num1,num2,fact, temp;
		
		System.out.print("Enter num1: ");
		num1 = sc.nextInt();
		
		System.out.print("Enter num2: ");
		num2 = sc.nextInt();
		System.out.println("Factorial of numbers from "+num1 + " to "+ num2);
		for(int i = num1 ; i<=num2; i++) {
				fact = 1;

				for(int j=1; j<=i; j++) {
					fact *= j;
				}
		
				System.out.println("Factorial of "+ i + ": "+ fact);
		}
	}
}
